### ISSUE TITLE
what shuold I do to make this program run all frame on  mai_city sequence 00?

### ISSUE THREAD

**Issue by Zhangjyhhh** [PRBonn/SHINE_mapping#15]
`setting:
  name: "maicity_incre_reg"
  output_root: "/home/hh/experiments/"
  pc_path: "/home/hh/data/mai_city/ply/sequences/01/velodyne"
  pose_path: /home/hh/data/mai_city/ply/sequences/01/poses.txt"
  calib_path: "/home/hh/data/mai_city/ply/sequences/01/calib.txt"
  load_model: False # load the pretrained decoder model (optional)
  model_path: "./pretrained/geo_decoder_8dim.pth" 
  first_frame_ref: False
  begin_frame: 0
  end_frame: 699
  every_frame: 1 # 1 means does not skip
  device: "cuda"
  gpu_id: "0"
process:
  min_range_m: 1.5
  pc_radius_m: 20.0  # distance filter for each frame
  rand_downsample: False # use random or voxel downsampling
  vox_down_m: 0.03
  rand_down_r: 0.2
sampler:
  surface_sample_range_m: 0.5
  surface_sample_n: 3
  free_sample_begin_ratio: 0.3
  free_sample_end_dist_m: 0.8
  free_sample_n: 3
octree:
  leaf_vox_size: 0.5
  tree_level_world: 12
  tree_level_feat: 3
  feature_dim: 8
  poly_int_on: True
  octree_from_surface_samples: True
decoder:
  mlp_level: 2
  mlp_hidden_dim: 32
  freeze_after_frame: 20
loss:
  ray_loss: False
  main_loss_type: sdf_bce  # select from sdf_bce (our proposed), sdf_l1, sdf_l2, dr, dr_neus
  sigma_sigmoid_m: 0.05
  loss_weight_on: False
  behind_dropoff_on: False
  ekional_loss_on: False
  weight_e: 0.1
continual:
  continual_learning_reg: True # using incremental mapping with regularization
  lambda_forget: 1e6 # the larger this value, the model would be less likely to forget
  window_replay_on: False # replay within the sliding window
  window_radius_m: 0
optimizer:
  iters: 50  # iterations per frame
  batch_size: 4096
  learning_rate: 0.01
  weight_decay: 0 # l2 regularization
eval:
  wandb_vis_on: False # log to wandb or not
  o3d_vis_on: True # visualize the mapping or not
  vis_freq_iters: 0 
  save_freq_iters: 0 # save the model and octree every x iterations
  mesh_freq_frame: 5 # reconstruct the mesh every x frames
  mc_res_m: 0.2 # reconstruction marching cubes resolution
  mc_with_octree: False # querying sdf in the map bbx
  mc_vis_level: 1
  save_map: False # save the sdf map or not`
thanks for your excellent work.
I just want to know that  why I change the parameter "end frame" from 100 to 699 in files config/maicity/maicity_incre_reg,  I get the same result when "end frame=100". I just want to run all frame in mai_city sequence 00.May need to modify another parameter too,could you tell me

**Comment by YuePanEdward** [comment]
Thanks for your interest in our work.
MaiCity sequence 01 only has 100 frames. You may need to change `pc_path` and `pose_path` to `xxx/sequences/00/xxx` to use MaiCity sequence 00, which contains 700 frames.
